docs/myriad-remote-distribution-configuration.md

```diff
@@ -0,0 +1,50 @@
+# Myriad Remote Distribution
+
+The Myriad Scheduler can be configured to automatically download and run the hadoop yarn binaries and get the hadoop 
+configuration from the resource manager. This means you won't have to install and configure hadoop yarn on each machine. 
+Note this is a very new feature and the configuration options may change dramatically in the future.
+
+# Myriad Remote Distribution Bundle Creation
+
+We will assume you are using hadoop-2.5.0 downloaded from hadoop.apache.org.  Specific vendor versions should work but 
+may require additional steps.  We will also assume hadoop is installed in `/opt/hadoop-2.5.0`, adjust this path to fit 
+your installation.
+
+First configure the Resource Manager as normal..
+
+At this point you build myriad with the commands:
+```Shell
+./gradlew build  
+./gradlew capsuleExecutor  
+```
+and copy the jars and config onto your yarn classpath:
+```Shell
+cp build/libs/*.jar /opt/hadoop-2.5.0/share/hadoop/yarn/lib/
+cp src/main/resources/myriad-config-default.xml
+```
+You will also need to place `myriad-executor-0.0.1.jar` in hdfs
+```Shell
+hadoop fs -put build/libs/myriad-executor-0.0.1.jar /dist
+```
+In the /opt/hadoop/etc/hadoop edit myriad-config-default.  For standard configuration see myriad-configuration(myriad-configuration.md).  To enable remote binaray distribution you must set the following options:
```

Comment:
Wrap "/opt/hadoop/etc/hadoop" in backticks so it looks like `/opt/hadoop/etc/hadoop`. Maybe just say "Edit the myriad configuration in `/opt/hadoop/etc/hadoop/myriad-config-default`"
Your link won't render unless you wrap the anchor text in square brackets like: `[myriad-configuration](myriad-configuration.md)`
s/binaray/binary/